src/liballoc/heap.rs

```diff
@@ -395,6 +425,12 @@ mod imp {
     }
 
     pub fn stats_print() {}
```

Comment:
Well I could directly `WriteConsole` to `GetStdHandle(STD_OUTPUT_HANDLE)` but then I'd have to format numbers into strings without `std`.